istio-control/istio-discovery/values.yaml

```diff
@@ -100,6 +100,10 @@ pilot:
   # Indicate if Galley is enabled to send MCP queries
   useMCP: true
 
+  # When true, DNS certificates will be provisioned through Chiron linked into Pilot.
+  # When false, such mechanism is not enabled.
+  enableCertificateController: false
```

Comment:
Move this to top level global, and rename to 'certificates:' 

In mesh config - add the content of this section using 'toJSON' - like we do for networks.

Reply:
And we want this enabled by default - so it gets tested. We can turn it off before the release. ( like we did with sniffing)

Reply:
I mean: enabled by default now, we'll decide to enable it or not before the release. We need to see what happens on K8S clusters that don't support the API.

Reply:
Done: addressed in https://github.com/istio/installer/pull/424; please review it.